compatibility/BUILD

```diff
@@ -52,13 +52,27 @@ config_setting(
     for sdk_version in sdk_versions
 ]
 
+# Instead of testing the full cartesian product of all SDK versions with
+# all platform (~= Sandbox/JSON API) versions, we test the latest version of
+# each with all versions of the other. This gives us a reasonable feedback
+# with regards to maintaining backwards-compatibility without causing the
+# test runs to grow quadratically.
+head = "0.0.0"
+
 [
     sdk_platform_test(
         platform_version = platform_version,
+        sdk_version = head,
+    )
+    for platform_version in platform_versions
+]
+
+[
+    sdk_platform_test(
```

Comment:
you now have 0.0.0 against 0.0.0 twice which fails with
```
ERROR: Traceback (most recent call last):
	File "/Users/vsts/agent/_work/1/s/compatibility/BUILD", line 71, column 22, in <toplevel>
		sdk_platform_test(
	File "/Users/vsts/agent/_work/1/s/compatibility/bazel_tools/testing.bzl", line 415, column 23, in sdk_platform_test
		client_server_test(
	File "/private/var/tmp/_bazel_vsts/6938b4f491cf5cda3150ff93fe52f44b/external/daml/bazel_tools/client_server/client_server_test.bzl", line 57, column 19, in client_server_test
		sh_inline_test(
	File "/private/var/tmp/_bazel_vsts/6938b4f491cf5cda3150ff93fe52f44b/external/daml/bazel_tools/sh/sh.bzl", line 54, column 22, in sh_inline_test
		_sh_inline_script(
Error in _sh_inline_script: _sh_inline_script rule 'ledger-api-test-tool-0.0.0-platform-0.0.0_script' in package '' conflicts with existing _sh_inline_script rule, defined at /Users/vsts/agent/_work/1/s/compatibility/BUILD:63:22
ERROR: package contains errors: 
ERROR: error loading package '': Package '' contains errors
```

Reply:
Yup, on it, thanks.

Reply:
I guess it does not make sense to test the latest platform and SDK together, that's not a compatibility test.

Reply:
I think it’s still a useful sanity check to make sure that the tests work as they should.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/8893/commits/c18bf26c0fe7c603e3f2efe407c89d5b39af968f